src/chrome/content/rules/arte.xml

```diff
@@ -62,21 +55,25 @@ Fetch error: http://info.arte.tv/ => https://info.arte.tv/: Too many redirects w
 		- tset
 		- wanted18
 
-	wrong content:
+	Wrong content:
 		- forums.arteradio.com
+
+	Redirect to HTTP:
+		- cinema.arte.tv
+		- future.arte.tv
+		- info.arte.tv
 -->
-<ruleset name="arte (partial)" platform="mixedcontent" default_off='failed ruleset test'>
+<ruleset name="arte (partial)" >
 	<target host="arte.tv" />
 	<target host="www.arte.tv" />
 	<target host="boutique.arte.tv" />
-	<target host="cinema.arte.tv" />
 	<target host="club.arte.tv" />
 	<target host="concert.arte.tv" />
-	<target host="future.arte.tv" />
-	<target host="info.arte.tv" />
+
 	<target host="arteradio.com" />
 	<target host="www.arteradio.com" />
 
+	<securecookie host=".+" name=".+" />
 	<rule from="^http:"
```

Comment:
No, add the line break between `securecookie` and `rule` tags and remove the one inside the `rule` tag.